### PR TITLE
Migrate usage of inventory to use v3

### PR DESCRIPTION
**Pull request by karelhala**
* [x] merge after https://github.com/RedHatInsights/frontend-components/pull/989

### CODE REVIEW

**Review by Hyperkid123 — COMMENTED** [review]
@karelhala can you bump the dependency version?

File: config/webpack.plugins.js [hunk 1]
```diff
@@ -28,6 +28,7 @@ const plugins = [
     shared: [
       { react: { singleton: true, requiredVersion: deps.react } },
       { 'react-dom': { singleton: true, requiredVersion: deps['react-dom'] } },
+      { 'react-redux': { singleton: true, requiredVersion: deps['react-redux'] } },
```
Comment (Hyperkid123) [inline config/webpack.plugins.js:31]: Is this required? It can actually break the dependencies at runtime. Hooks will be referencing to incorrect store.

Comment (karelhala) [reply config/webpack.plugins.js:31]: Yup, inventory is using react-redux functions and without this it trhew an error with missing dependency. We could remove the singleton here

Comment (Hyperkid123) [reply config/webpack.plugins.js:31]: Yeah we have to remove the singleton

**Review by Hyperkid123 — APPROVED** [review]

**Pull request merged**